packages/sdk-js/src/GrowthBook.ts

```diff
@@ -894,7 +937,29 @@ export class GrowthBook<
     else console.log(msg, ctx);
   }
 
+  public enqueueTrackingCalls(
+    data: TrackExperimentData | TrackExperimentData[]
+  ) {
+    data = Array.isArray(data) ? data : [data];
```

Comment:
should we bother with this, or just be opinionated about accepting an array only?